lms/envs/common.py

```diff
@@ -516,7 +516,15 @@
     # .. toggle_tickets: https://github.com/edx/edx-platform/pull/9744
     'ENABLE_SPECIAL_EXAMS': False,
 
-    # Enable OpenBadge support. See the BADGR_* settings later in this file.
+    # .. toggle_name: FEATURES['ENABLE_OPENBADGES']
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: Enables support for the creation of OpenBadges as a method of awarding credentials.
+    # .. toggle_warnings: The following settings (all of which are in this file) should be set or reviewed prior to
```

Comment:
I would either remove "(all of which are in this file)", or change it to "(all of which are in the same file)", so it will read correctly when published outside of the code here: https://edx.readthedocs.io/projects/edx-platform-technical/en/latest/

Reply:
Note: This wasn’t updated. Do you want to, or should I merge as-is?

Reply:
Fixed. Should be good to go now. :)